src/App.tsx

```diff
@@ -10,33 +12,25 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const onTabSelected = (tab: Tab | undefined) => {
+    if (tab) {
+      setSelectedTabId(tab.id);
+    }
```

Comment:
It is recommended to use a default value instead of checking if the tab object is undefined.